src/main/java/com/ucl/imaginethisserver/Model/Feedback.java

```diff
@@ -0,0 +1,78 @@
+package com.ucl.imaginethisserver.Model;
+
+import java.sql.Timestamp;
+import java.util.UUID;
+
+public class Feedback {
+    private final UUID feedbackID;
+    private final UUID userID;
+    private final String projectID;
+    private int upvotes;
+    private int downvotes;
+    private String userName;
+    private Timestamp timestamp;
+    private String text;
+
+    public Feedback(UUID feedbackID, String projectID, int upvotes, int downvotes, UUID userID, String userName, Timestamp timestamp, String text) {
```

Comment:
And I'd add another overloaded constructor with removed `upvotes` and `downvotes` arguments. This will be used when constructing a new feedback via the `addNewFeedback`, as I think it could be confusing having to specify `0, 0` when adding a new feedback.